DEVELOPING.md

```diff
@@ -0,0 +1,75 @@
+# Gettting Started
+
+## Prerequisites
+
+You will want to download and install the following:
+
+* [Vagrant](https://www.vagrantup.com/downloads.html)
+* [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+* [VirtualBox Extension Pack](https://www.virtualbox.org/wiki/Downloads): select the version that matches your version of VirtualBox
+* [git](https://git-scm.com/downloads)
+
+**Note**: You may also need go into your BIOS to ensure that VT-x/AMD-v (also known as *Intel(R) Virtualization Technology*) settings are enabled.
```

Comment:
They may also need to add `vagrant` to their path.

Reply:
Seems like it's [part of the installer](https://www.vagrantup.com/docs/installation/) now, so I just added a note about possibly logging out.